packages/plugins/connections/connection-mongodb/src/connections/MongoDBCollection/getCollection.js

```diff
@@ -14,7 +14,8 @@
   limitations under the License.
 */
 
-import { MongoClient } from 'mongodb';
```

Comment:
Do we get an error if we import like this?

Reply:
Yes we get a named export not found when imported in the build package.
`SyntaxError: Named export 'MongoClient' not found. The requested module 'mongodb' is a CommonJS module, which may not support all module.exports as named exports.`

Reply:
@SamTolmay This will not be a problem in the new major version of MongoDB driver. It already uses named exports.